scripts/upload.py

```diff
@@ -64,7 +63,13 @@ def upload(globpath: str, container: str, queue: str, storage_account_uri: str):
             upload_succeded = False
             with open(infile, "rb") as data:
                 try:
-                    retry_on_exception(lambda: blob_client.upload_blob(data, blob_type="BlockBlob", content_settings=ContentSettings(content_type="application/json")), raise_exceptions=[ResourceExistsError])
+                    def _upload():
```

Comment:
Is this just a stylistic/readability thing or something more? I like the change either way.

Reply:
Yes and no, the line was really long before and there was a linter error on the `blob_client.upload_data` as it did not have proper type hints, so I did this so that I could have a targeted linter suppression on that line.